server/iframe.go

```diff
@@ -83,7 +83,7 @@ var iFrameHTML = `<!DOCTYPE html>
 </head>
 <body>
 	<iframe
-		style="position:absolute;top:0px;width:100%;height:100vh;"
+		style="position:absolute;top:0;left:0;width:100%;height:100%;border=none;"
```

Comment:
minor readability preference for CSS and needed to change `border=none` to `border: none`
```suggestion
		style="position: absolute; top: 0; left: 0; width: 100%; height: 100%; border: none;"
```